src/core/selection/state/selection.state.js

```diff
@@ -118,6 +122,79 @@ export default class SelectionState {
 		return item;
 	}
 
+	selectByKeys(keys, unit) {
+		if (keys.length === 0) {
+			return;
+		}
+
+		if (isUndefined(unit)) {
+			unit = this.model.selection().unit;
+		}
+		const data = this.model.data();
+
+		switch (unit) {
+			case 'row': {
+				const rows = data.rows;
+				rows.forEach(row => {
+					const rowKey = this.key(row);
+					const found = keys.indexOf(rowKey) > -1;
+					if (found) {
+						this.select(row);
+					}
+				});
+				break;
+			}
+			case 'column': {
+				const columns = data.columns;
+				columns.forEach(column => {
+					const colKey = this.key(column);
+					const found = keys.indexOf(colKey) > -1;
+					if (found) {
+						this.select(column);
+					}
+				});
+				break;
+			}
+			case 'cell': {
+				const cells = [];
+				data.columns.forEach(column => {
+					data.rows.forEach(row => {
+						cells.push({
+							column: column,
+							row: row
+						});
+					});
+				});
+				cells.forEach(cell => {
+					const cellKey = this.key(cell);
+					const found = keys.findIndex(key => `${key.column}[${key.row}]` == cellKey) > -1;
```

Comment:
because if it was smth like key => this.key(key) === cellKey it wasn't strange :)
btw please use ===